assignments/callbacks.js

```diff
@@ -2,27 +2,49 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function firstItem(arr, cb) {
   // firstItem passes the first item of the given array to the callback function.
+  if(typeOf === "function"){
```

Comment:
I like that you thought to check to make sure that it's a function. I think you need it to be `typeof cb === 'function'` to check the cb.